pages/playlist/[id].js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import viewPlaylistDetails from '../../api/mergedData';
+import PlaylistForm from '../../components/forms/PlaylistForm';
+
+export default function ViewPlaylists() {
+  const [editItem, setEditItem] = useState({});
+  const router = useRouter();
+
+  const { id } = router.query;
+
+  useEffect(() => {
+    viewPlaylistDetails(id).then(setEditItem);
```

Comment:
When I run this on my local machine, I get `TypeError: (0 , _api_mergedData__WEBPACK_IMPORTED_MODULE_3__.default) is not a function`

Reply:
I think we solved this issue last night